Mainframe3270/keywords/assertions.py

```diff
@@ -1,8 +1,10 @@
 import re
 from typing import List, Optional
+
```

Comment:
@samuelpcabral , I am sorry, I know you removed these line breaks, but they are inserted automatically by `isort` and I haven't yet figured out how to prevent the tool from doing this.

I hope this is not an issue